contentcuration/contentcuration/static/js/edit_channel/views.js

```diff
@@ -188,6 +189,10 @@ var BaseView = Backbone.View.extend({
 				if(model.id === window.current_user.get('clipboard_tree').id){
 					window.current_user.set('clipboard_tree', model.toJSON());
 				}
+
+				if(model.id === window.current_channel.get("trash_tree").id) {
```

Comment:
When items are being reloaded, update the trash tree if it's part of the ancestors. Otherwise, when items are moved out of the trash tree via the Trash modal, it will still show them when you reopen the modal (legacy children)